### PR TITLE
Add null check for link before usage

### PR DESCRIPTION
**Pull request by tomjw64**
Resolves https://github.com/preactjs/preact-iso/issues/46

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -16,10 +16,13 @@ const UPDATE = (state, url) => {
 			return state;
 		}
 
-		const link = url.target.closest('a[href]'),
-			href = link.getAttribute('href');
```
Comment (rschristian) [inline src/router.js:20]: Should just do this instead:

```js
const link = url.target.closest('a[href]'),
	href = link && link.getAttribute('href');
```

Comment (tomjw64) [reply src/router.js:20]: Sure, no problem

Comment (tomjw64) [reply src/router.js]: Done! :)

**Review by rschristian — APPROVED** [review]
Thanks! I had spotted something last night which made me suspect an issue, but hadn't had the chance to check yet.

Will push a patch in the next couple of minutes.

**Pull request merged**